index.html

```diff
@@ -0,0 +1,43 @@
+<!DKunkel pr 1 -On Paper>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css"
+  </head>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Courgette&family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&display=swap"
+    rel="stylesheet"
+  />
+
+  <body>
+    <h1>on paper</h1>
+    <h2>by Jacqueline Woodson</h2>
+
+    <p>The first time I write my full name</p>
+
+    <p><em>Jacqueline Amanda Woodson</em></p>
+
+    <p>without anybody's help</p>
+    <p>on a clean white page in my composition notebook,</p>
+    <p>I know</p>
+
+    <p>if I wanted to</p>
+
+    <p>
+      <em>I could write <strong>anything</strong></em>
+    </p>
+
+    <p>Letters becoming words, words gathering meaning</p>
+    <p>becoming</p>
+    <p>thoughts <em>outside</em> my head</p>
+
+    <p>becoming sentences</p>
+
+    <p>written by</p>
+
+    <p><em>Jacqueline Amanda Woodson</em></p>
+  </body>
+</html>
```

Comment:
Other than the couple of tweaks to the content in your HTML `head`, the rest of this HTML is spot on and you've clearly made choices that communicate things about how I would read this poem aloud. GREAT WORK 🏆🌈